### PR TITLE
New home: service catalog

### PR DESCRIPTION
**Pull request by stefanalund**
## Hey, I just made a Pull Request!

This PR changes the home/landing page to the Service Catalog. It also introduces the `DismissablePanel` component, which currently lacks support for being permanently dimissed.

Screenshot 2020-05-26 at 20 37 00

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [x] Prettier run on changed files
- [ ] Tests added for new functionality

### CODE REVIEW

File: packages/core/src/components/DismissableBanner/DismissableBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+ */
+
+import React, { FC, ReactNode, useState } from 'react';
+import classNames from 'classnames';
+import { makeStyles, Theme } from '@material-ui/core';
+import Snackbar from '@material-ui/core/Snackbar';
+import SnackbarContent from '@material-ui/core/SnackbarContent';
+import IconButton from '@material-ui/core/IconButton';
+import Close from '@material-ui/icons/Close';
+// import { useSetting, Setting } from 'shared/apis/settings';
```
Comment (stefanalund) [inline packages/core/src/components/DismissableBanner/DismissableBanner.tsx:24]: These needs to be brought over from internal Backstage.

File: packages/core/src/components/DismissableBanner/DismissableBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    position: 'relative',
```
Comment (Rugvip) [inline packages/core/src/components/DismissableBanner/DismissableBanner.tsx:28]: Is this and zIndex really needed?

File: packages/core/src/components/DismissableBanner/DismissableBanner.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    />
+  </div>
+);
+
+export const WithLink = () => (
+  <div style={containerStyle}>
+    <DismissableBanner
+      message={
+        <>
+          <span>This is a dismissable banner with a link: </span>
```
Comment (Rugvip) [inline packages/core/src/components/DismissableBanner/DismissableBanner.stories.tsx:57]: Doesn't have proper spacing between the text and the link, I think you want to put the Link inside the span?

**Review by Rugvip — APPROVED** [review]
Some minor comments, but 👍

File: packages/core/src/components/DismissableBanner/DismissableBanner.tsx (hunk 2 above)
Comment (stefanalund) [reply packages/core/src/components/DismissableBanner/DismissableBanner.tsx:28]: Turns out, yes, relative is needed. But not zIndex.

Screenshot 2020-05-28 at 11 56 42

File: packages/core/src/components/DismissableBanner/DismissableBanner.stories.tsx (hunk 3 above)
Comment (stefanalund) [reply packages/core/src/components/DismissableBanner/DismissableBanner.stories.tsx]: Fixed.

**Pull request merged**